deploy/README.md

```diff
@@ -45,6 +45,50 @@ To setup `lvmd`:
 
 3. Install `lvmd` and `lvmd.service`, then start the service.
 
+### OPTIONAL: Launch lvmd with DaemonSet
```

Comment:
Could you describe the top of the section that lvmd can be deployed as a daemon or using daemonset?

Reply:
Excuse me. I updated the lvmd description in the below commit:
https://github.com/topolvm/topolvm/pull/285/commits/aeef54c190f709d11c3d517701eaec6ae6f535a7
Is the issue that you commented on resolved?

Reply:
Yes, it's fine.